lib/analyzer/static-analyzer.ts

```diff
@@ -82,38 +81,41 @@ export async function analyze(
     );
   }
 
-  const dockerArchive = await getDockerArchiveLayersAndManifest(
+  const {
+    imageId,
+    manifestLayers,
+    extractedLayers,
+  } = await archiveExtractor.getArchiveLayersAndManifest(
```

Comment:
I realise this returns more than the layers and manifest but I'm happy with the name for now, if we find that we're returning even more data in the future then it will get a slight rename I think!

Reply:
Yes is does, but I just didn't know what would be a better name 😆 Because it just essentially returns extraction result which is layers, some info from the manifest and imageId. I think I'm fine with it for now too as it's still related to manifest data & layers, sort of